src/Classes/MyRadio/CoreUtils.php

```diff
@@ -554,6 +554,34 @@ public static function getAllPermissions()
     }
 
     /**
+     * udiff function for permission value equality
+     * @param  array $perm1 permission value & description
+     * @param  array $perm2 permission value & description
+     * @return int          comparison result
+     */
+    private static function comparePermission($perm1, $perm2)
```

Comment:
Yeah, until that is merged stuff should still go in coreutils and separated out in that PR